### PR TITLE
add reliableGPT example to OpenAI Cookbook (third party examples)

### PR DESCRIPTION
**Pull request by ishaan-jaff**
this PR adds an example of using reliableGPT (https://github.com/BerriAI/reliableGPT/ ) to make batch calls to OpenAI. 
- it handles rate limits, token limits and optimizes for parallelizing requests to remain under the rate limits. 
- With reliableGPT developers can make 200 GPT-4 requests to OpenAI and are guaranteed a response for each call

### CODE REVIEW

**Comment by ishaan-jaff** [comment]
cc @logankilpatrick @ted-at-openai @shyamal-anadkat

**Comment by ted-at-openai** [comment]
I will take a look this week. Thanks for your patience!

File: examples/third_party_examples/ReliableGPT_Getting_Started.ipynb [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+          "text": [
+            "Hello! How can I assist you today?\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title ## 🤯 Running 200 GPT-4 OpenAI Calls at once!\n",
```
Comment (ted-at-openai) [inline examples/third_party_examples/ReliableGPT_Getting_Started.ipynb:68]: might want to add a warning about how much it's going to cost each you run this cell - is it like a dollar?

Comment (ted-at-openai) [inline examples/third_party_examples/ReliableGPT_Getting_Started.ipynb:68]: why does this say 200 calls? I see `range(20)` along with a messages that contains a super long string thanks to the `*400`.

**Pull request closed without merging**